lib/db/mysql.js

```diff
@@ -308,79 +308,73 @@ module.exports = function (log, error) {
     )
   }
 
-  const ADD_CAPABILITY = 'CALL addCapability_1(?, ?, ?)'
-  const PURGE_CAPABILITIES = 'CALL purgeCapabilities_1(?, ?)'
+  const UPSERT_AVAILABLE_COMMAND = 'CALL upsertAvailableCommand_1(?, ?, ?, ?)'
+  const PURGE_AVAILABLE_COMMANDS = 'CALL purgeAvailableCommands_1(?, ?)'
 
   var CREATE_DEVICE = 'CALL createDevice_4(?, ?, ?, ?, ?, ?, ?, ?, ?)'
```

Comment:
I didn't spot this when it happened before, but can you move this definition of `CREATE_DEVICE` down so it's next to the `createDevice` method again?